app-frontend/src/store/sagas/worker.ts

```diff
@@ -38,3 +46,32 @@ export function* loadWorkerData(action: WorkerIdAction) {
     }
   }
 }
+
+export function* addWorkerData(action: WorkerAddAction) {
+  try {
+    axios({
+      method: 'POST',
+      url: `${API_URL}/workers`,
+      data: action.payload,
+      withCredentials: true,
+    });
+  } catch (error: unknown) {
+    if (error instanceof Error) {
```

Comment:
You shouldn't have here check on instanceof Error. You must handle here any kind of error, and show an appropriate message to user.